install/install_requirements.sh

```diff
@@ -74,16 +74,28 @@ then
 elif [[ -x "$(command -v rocminfo)" ]];
 then
   TORCH_NIGHTLY_URL="https://download.pytorch.org/whl/nightly/rocm6.2"
+elif [[ -x "$(command -v xpu-smi)" ]];
+then
+  TORCH_NIGHTLY_URL="https://download.pytorch.org/whl/nightly/xpu"
 else
   TORCH_NIGHTLY_URL="https://download.pytorch.org/whl/nightly/cpu"
 fi
 
 # pip packages needed by exir.
-REQUIREMENTS_TO_INSTALL=(
-  torch=="2.6.0.${PYTORCH_NIGHTLY_VERSION}"
-  torchvision=="0.22.0.${VISION_NIGHTLY_VERSION}"
-  torchtune=="0.5.0.${TUNE_NIGHTLY_VERSION}"
-)
+if [[ -x "$(command -v xpu-smi)" ]];
+then
+  REQUIREMENTS_TO_INSTALL=(
+    torch=="2.6.0.${PYTORCH_NIGHTLY_VERSION}"
+    torchvision=="0.22.0.${VISION_NIGHTLY_VERSION}"
+    torchtune=="0.4.0"
```

Comment:
Hi @songhappy, are we able to bump this to 0.5.0?

I also noticed that over in tune https://github.com/pytorch/torchtune/pull/1280, another intel eng ([Ankur-singh](https://github.com/Ankur-singh)) is checking in on nightly stsatus

Reply:
Hi @Jack-Khuu , I have updated torchtune version to 0.5.0. Update to now, nightly version is still unavailable. Can you review this PR and check if it can be merged. If the torchtune nightly on XPU is ready, we can update the installation later in a new PR.